packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -55,7 +55,6 @@ Requires:       upstart
 Requires:       iptables
 Requires:       docker >= 17.06.2ce
 Requires:       procps
-Requires:       dhclient
```

Comment:
At the bottom of the spec, in the `%files` section, there is a directive that still refers to the removed path, can you remove that too please?

This specifically:

`%ghost %{_sharedstatedir}/ecs/dhclient`

There's also a `mkdir` that was creating the directory that doesn't need to be there any more.